Source/Core/Logging/FramesLogEvent.swift

```diff
@@ -14,10 +15,32 @@ enum FramesLogEvent: Equatable {
         case serverError
         case tokenID
         case tokenType
+        case locale
+        case theme
+        case primaryBackgroundColor
```

Comment:
I am expecting these will be thrown away as with the new UI release 4.0.0 these will be deleted anyways.